version.ts

```diff
@@ -0,0 +1 @@
+export const VERSION = "0.32.0";
```

Comment:
This adds a bit more maintenance that I don't want to do at the moment and I'm not sure how useful it is. I think once https://github.com/dsherret/dax/tree/distribute_bundled is merged then I can easily do this.